splinterd/tests/admin/payload.rs

```diff
@@ -236,3 +238,25 @@ fn setup_circuit(
         .into_proto()
         .expect("Unable to get proto from `CreateCircuit`")
 }
+
+/// Create the bytes of a `CreateContractRegistryAction` batch
+pub(in crate::admin) fn make_create_contract_registry_batch(
+    name: &str,
+    signer: &dyn Signer,
+) -> Batch {
+    let owners = vec![signer
+        .public_key()
+        .expect("Unable to get signer's public key")
+        .as_hex()];
+    CreateContractRegistryActionBuilder::new()
+        .with_name(name.into())
+        .with_owners(owners)
+        .into_payload_builder()
+        .expect("Unable to create payload builder for `CreateContractRegistryActionBuilder`")
```

Comment:
I think that we should probably propagate these as InternalError, and have the `expect` done by the caller.

Reply:
Ok, updated these to propagate these as `InternalError`